docs/helm/helm_reset.md

```diff
@@ -6,7 +6,7 @@ uninstalls Tiller from a cluster
 
 
 
-This command uninstalls Tiller (the helm server side component) from your
+This command uninstalls Tiller (the helm server-side component) from your
```

Comment:
- [ ] While we're still pursuing these corrections, why is "helm" written in lowercase here?  
  Did we miss that in the _reset.go_ file?